packages/babel-plugin-transform-new-target/test/fixtures/general/class-properties/options.json

```diff
@@ -2,6 +2,6 @@
   "plugins": [
     "transform-new-target",
     "transform-arrow-functions",
-    ["proposal-class-properties", { "loose": true }]
+    ["proposal-class-properties", { "loose": false }]
```

Comment:
Should we keep this and create a new test for loose: false behavior?

Reply:
Sure! I changed loose to make it more readable, and forgot to change it back😭